app/decorators/application_choice_export_decorator.rb

```diff
@@ -35,6 +35,13 @@ def nationalities
       .sort.partition { |e| %w[GB IE].include? e }.flatten
   end
 
+  def nationality
```

Comment:
Logic looks good — but shouldn't this live in the `HESADataExport` as the rule is HESA-specific? I _think_ we still want to return multiple nationalities in the ordinary export.

Reply:
We do. However as the decorator exposes fields that are specific to the ApplicationDataExport service only but is used both there and in the HESADataExport I assumed that it was purposely generic enough to be used by both services. 

HESADataExport does not seem like a good fit as only `call` is - and should be public - which means I cannot directly test the logic behind identifying the correct nationality.

I think a decorator specific to the HESA export would probably be the best approach.